srv/ResetSimulation.srv

```diff
@@ -0,0 +1,15 @@
+# Reset the simulation to the start, including the entire scene and the simulation time.
```

Comment:
Another idea here, which I have had to tinker with, is the difference between:

- Resetting back to the exact original state

vs. 

- Resetting back to a valid initial state by re-sampling spawn poses from their appropriate distributions.

May be worth a distinction in the enum?

Reply:
Definitely interesting applications for RL! I would imagine randomization as a separate feature which will control the behavior of STATE reset: If a flag is on somewhere, reset also randomizes according to parametrization.

I think it is enough to have one field, but I added a clarification as you suggested.